default_configs/alertmanager.yml

```diff
@@ -20,4 +20,5 @@ route:
   repeat_interval: 1h
 receivers:
 - name: null_receiver
+- name: default_test
```

Comment:
Huh, what's this?

Reply:
whoops, accidentally left a change made from running it